html-css/week1/Portfolio/index.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Neepun Raj Shrestha</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+  </head>
+  <body>
+   
+    <header class="main-header container " id="hero">
+        <div class="logo" >
+          <img src="./img/logo.png" alt="Logo" class="logo-img" />
+        </div>
+        <nav class="nav-header">
+          <ul class="nav-list">
+            <li><a href="#hero">Home</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#projects">Projects</a></li>
+            <li><a href="#contacts">Contact</a></li>
+          </ul>
+        </nav>
+      </header>
+    <main>
+        
+      <section class="hero container ">
+        <h1>Transform <br />Your Idea into <br />Action</h1>
+        <p>Hi, I am Neepun Raj your next Software Engineer</p>
+        <h2>Full Stack Developer</h2>
+      </section>
+
+      <section class="about" id="about">
+        <h2 class="about__heading section-heading">About</h2>
+        <div class="about__content">
+            <!-- div 1  profile and pic-->
+            <div class="profile">
+                <div class="profile__picture">
+                    <img src="./img/neepun.jpg" alt="Neepun Raj"  />
+                </div>
```

Comment:
Could you move the class name down to the img tag and remove the `<div>` tag?

```suggestion
               <img src="./img/neepun.jpg" alt="Neepun Raj"  class="profile__picture"  />
```